loconomics/loconomics
# is437 skip payment info

This PR is for #437. 

When I implemented skipping payments, I realized that two critical features were missing. I should have created separate issues for them, but bundled them into this issue anyway:
1. On the listings dashboard pages, always show the _Preview Listing_ buttons, even if the listing is inactive
2. When a professional is viewing their own profile, at the top of the profile is a new message bar indicating whether the listing is active or inactive. It also has an _Edit_ button linking back to the listing dashboard page.

To test:
1. Verify that you can finish professional onboarding by skipping the payment and by not skipping the payment account.
  a. the _Skip for now_ button on the payment account form should only be visible during onboarding.
2. When you have completed onboarding, you should be able to preview your own listing, even if it's inactive.
3. On your profile, you should be able to click around to all of your job titles, and the active/inactive status in the message bar should update.
4. If you view a profile that is not your own, or if you are not logged in, you should _not_ see the active/inactive bar.
5. If you view a profile that is not your own, you should only see _active_ profiles.
